train.py

```diff
@@ -232,6 +232,12 @@ def loss_fn(pred, labels):
     model_config.max_seq_len = job_config.training.seq_len
 
     logger.info(f"Building {model_name} {job_config.model.flavor} with {model_config}")
+    logger.info(
+        f"Detailed training config, local_batch_size: {job_config.training.batch_size}, "
```

Comment:
let's remove underscore as they do not always refer to variable name, e.g. use `local batch size`, `sequence length`, `total steps`